packages/manager/apps/web/client/app/domain/zone/domain-zone-dns.controller.js

```diff
@@ -93,6 +95,49 @@ export default class DomainTabZoneDnsCtrl {
     this.$scope.$broadcast('paginationServerSide.loadPage', 1);
   }
 
+  /**
+   * Checks if there's a localStorage item indicating a recent DNS zone deletion and triggers an alert.
+   *
+   * This function looks for a specific item in the localStorage, identified by a key that combines
+   * a base string ('dns-delete-') with the domain name. The value of this item should be a date
+   * representing when a DNS zone deletion was performed. If this date is within the last 72 hours,
+   * an alert message is displayed. If the date is older than 72 hours, the item is removed from
+   * localStorage, assuming the alert is no longer relevant.
+   */
+  checkAlertZoneDeleteMessage() {
+    // Construct the localStorage key using the domain name
+    const localStorageKey = `dns-delete-${this.domain.name}`;
+
+    // Retrieve the stored value (date) from localStorage
+    const storedValue = window.localStorage.getItem(localStorageKey);
+
+    // Proceed only if a value was found
+    if (storedValue) {
+      // Parse the stored date using moment.js
+      const storedDate = moment(storedValue);
+      const currentDate = moment();
+
+      // Calculate the difference in hours between the current date and the stored date
+      const hoursDifference = currentDate.diff(storedDate, 'hours');
```

Comment:
```suggestion
      const storedDate = new Date(storedValue);
      const currentDate = new Date();

      // Calculate the difference in hours between the current date and the stored date
      const hoursDifference = Math.abs((currentDate - storedDate) / (1000 * 60 * 60));
```

Reply:
Thanks for the suggestion ! 
We use moment.js across the project for consistency and readability. 
While native JavaScript is efficient, sticking with moment.js aligns with the current codebase and simplifies maintenance.

Reply:
We try to remove time after time moment.js this is why I am proposing this change :)

Reply:
Thank you for the clarification regarding the gradual phase-out of moment.js from the project. 
Based on this perspective, your suggestion to use native JavaScript for date handling makes sense. 
I've implemented the change and pushed the update. 
Thanks for your guidance!

Reply:
@qpavy Additionally, I would appreciate it if you could take a moment to review the latest changes. 
If everything looks good and aligns with the project's direction for removing moment.js, could you please mark the suggested change as resolved? 
This would help in maintaining a clear status of the PR. 
Thank you for your support and collaboration!